lib/services/messenger/translators/translator-scaffold.ts

```diff
@@ -74,10 +74,10 @@ export abstract class TranslatorScaffold implements Translator {
 				const charRegex = new RegExp(`^${charCapture}`, 'im');
 				return TranslatorScaffold.metadataByRegex(message, charRegex);
 			case MetadataEncoding.HiddenHTML:
-				const hiddenHTMLRegex = new RegExp(`^<a href="${baseUrl}${querystring}"[^>]*></a>`, 'im');
+				const hiddenHTMLRegex = new RegExp(`<a href="${baseUrl}${querystring}"[^>]*></a>`, 'im');
 				return TranslatorScaffold.metadataByRegex(message, hiddenHTMLRegex);
 			case MetadataEncoding.HiddenMD:
-				const hiddenMDRegex = new RegExp(`^\\[\\]\\(${baseUrl}${querystring}\\) `, 'im');
+				const hiddenMDRegex = new RegExp(`\\[\\]\\(${baseUrl}${querystring}\\)`, 'im');
```

Comment:
These two mean now that I can post a message and (accidentally/intentionally) include some text that's interpreted as metadata in my own message. Does that have security/awkward bug repercussions?

Reply:
Okay, so the email that Front emits (checked against the 'show original'), does not simply end with </a>, so we can be much more discerning by checking that this is at the actual end of the string.  Just committing this wisdom now.